features/shop/cart_with_different_taxes_for_different_zones.feature

```diff
@@ -0,0 +1,50 @@
+@ui-cart
+Feature: Apply taxes based on zone and tax rate
+    In order to buy goods with correct taxes applied
+    As a Customer
+    I want to have correct taxes applied to my order
+
+    Background:
+        Given the store is operating on a single "France" channel
+        And default currency is "EUR"
+        And default tax zone is "EU"
+        And store has "EU VAT" tax rate of 23% for "Clothes" within "EU" zone
+        And store has "No tax" tax rate of 0% for "Clothes" for the rest of the world
+        And store has "Low VAT" tax rate of 5% for "Mugs" for the rest of the world
+        And store has a product "PHP T-Shirt" priced at "€100.00"
+        And product "PHP T-Shirt" belongs to "Clothes" tax category
+        And store has a product "Symfony Mug" priced at "€50.00"
+        And product "Symfony Mug" belongs to "Mugs" tax category
+        And store ships everything for free
+        And store allows paying offline
+        And I am logged in customer
+
+    Scenario: Displaying correct tax before specifying shipping address
+        When I added product "PHP T-Shirt" to the cart
```

Comment:
Added suggests it is a pre condition, I think it should be action (When) -> ``add product``